core/arch/arm/sm/sm_a32.S

```diff
@@ -139,9 +139,23 @@ UNWIND(	.cantunwind)
 	orr	r0, r0, #(SCR_NS | SCR_FIQ) /* Set NS and FIQ bit in SCR */
 	write_scr r0
 
+	dsb
```

Comment:
Hmmm, now that I think about it, I'm not sure, I'll remove them and see if they had a purpose.